ibm_catalog.json

```diff
@@ -0,0 +1,311 @@
+                },
+                {
+                  "dependency_output": "log_analysis_ingestion_key",
+                  "version_input": "log_analysis_ingestion_key"
+                },
+                {
+                  "dependency_input": "cloud_monitoring_provision",
+                  "version_input": "cloud_monitoring_enabled"
+                },
+                {
+                  "dependency_output": "cloud_monitoring_access_key",
+                  "version_input": "cloud_monitoring_access_key"
+                },
+                {
+                  "dependency_input": "region",
+                  "version_input": "cloud_monitoring_instance_region"
+                }
+              ]
+            }
+          ],
+          "dependency_version_2": true,
+          "configuration": [
+            {
+              "key": "cloud_monitoring_access_key",
+              "required": true
+            },
+            {
+              "key": "cloud_monitoring_add_cluster_name",
+              "required": false
+            },
+            {
+              "key": "cloud_monitoring_agent_name",
+              "required": false
+            },
+            {
+              "key": "cloud_monitoring_agent_namespace",
+              "required": false
+            },
+            {
+              "key": "cloud_monitoring_agent_tags",
+              "required": false
+            },
+            {
+              "key": "cloud_monitoring_agent_tolerations",
+              "required": false
+            },
+            {
+              "key": "cloud_monitoring_enabled",
+              "required": true
+            },
+            {
+              "key": "cloud_monitoring_endpoint_type",
+              "required": false
+            },
+            {
+              "key": "cloud_monitoring_instance_region",
+              "display_name": "Region",
+              "required": true,
+              "custom_config": {
+                "type": "vpc_region",
+                "grouping": "deployment",
+                "original_grouping": "deployment",
+                "config_constraints": {
+                  "generationType": "2"
+                }
+              }
+            },
+            {
+              "key": "cloud_monitoring_metrics_filter",
+              "required": false
+            },
+            {
+              "key": "cloud_monitoring_secret_name",
+              "required": false
+            },
+            {
+              "key": "cluster_config_endpoint_type",
+              "required": false
+            },
+            {
+              "key": "cluster_id",
+              "required": true
+            },
+            {
+              "key": "cluster_resource_group_id",
+              "required": true
+            },
+            {
+              "key": "cluster_data",
+              "required": true
+            },
+            {
+              "key": "ibmcloud_api_key",
```

Comment:
This would usually be the first input - can you review the ordering of the vars pleasE?